C/plugins/storage/sqlite/connection.cpp

```diff
@@ -39,6 +39,7 @@ using namespace rapidjson;
 
 #define F_TIMEH24_S     "%H:%M:%S"
 #define F_DATEH24_S     "%Y-%m-%d %H:%M:%S"
+#define F_DATEH24_M     "%Y-%m-%d %H:%M"
```

Comment:
Is this pull request incomplete? I would have expected to see this #define being used.

Reply:
Yes @MarkRiddoch , missing code is added now